code/modules/martial_arts/krav_maga.dm

```diff
@@ -125,7 +132,9 @@ datum/martial_art/krav_maga/grab_act(var/mob/living/carbon/human/A, var/mob/livi
 					  "<span class='userdanger'>[A] [picked_hit_type] you!</span>")
 	return 1
 
-/datum/martial_art/krav_maga/disarm_act(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+/datum/martial_art/krav_maga/disarm_act(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D, var/mob/living/carbon/human/H)
+	if(H.mind.assigned_role == "Head of Security") // De este modo el hos no adquiere el 60% de desarme del krav maga.
+		return
```

Comment:
En este caso deberias poner return FALSE (o return 0)

Reply:
Yea got it